tests/browsercontext-route.spec.ts

```diff
@@ -235,3 +235,122 @@ it('should overwrite post body with empty string', async ({ context, server, pag
   const body = (await req.postBody).toString();
   expect(body).toBe('');
 });
+
```

Comment:
NB: I'll re-organize the tests at the end before flipping to from draft. They'll likely end up their own file or next to the existing tests they relate (e.g. HAR).